inlineplz/interfaces/github.py

```diff
@@ -67,16 +68,19 @@ def is_duplicate(self, message, position):
         return False
 
     @staticmethod
-    def format_message(message):
+    def format_message(message, prefix):
         if not message.comments:
             return ''
         if len(message.comments) > 1:
-            return (
-                '```\n' +
-                '\n'.join(sorted(list(message.comments))) +
-                '\n```'
-            )
-        return '`{0}`'.format(list(message.comments)[0].strip())
+            comment_output = '\n'.join(sorted(list(message.comments)))
+            if prefix:
+                comment_output = prefix + '\n' + comment_output
+            return '```\n {0} \n ```'.format(comment_output)
+
+        comment_output = list(message.comments)[0].strip()
```

Comment:
i think you'd be better off just getting rid of the single-line use case entirely. i was just doing it because single line formatting with triple backticks looks a little odd, but if we're going to be adding a prefix on a 2nd line anyway, may as well always use triple backticks.